src/Compilers/CSharp/Portable/FlowAnalysis/EmptyStructTypeCache.cs

```diff
@@ -213,6 +213,13 @@ private FieldSymbol GetActualField(Symbol member, NamedTypeSymbol type)
                 case SymbolKind.Event:
                     var eventSymbol = (EventSymbol)member;
                     return (!eventSymbol.HasAssociatedField || ShouldIgnoreStructField(eventSymbol, eventSymbol.Type)) ? null : eventSymbol.AssociatedField.AsMember(type);
```

Comment:
>AssociatedField [](http://example.com/codeflow?start=140&length=15)

We should look for other similar places in the code base. I.e. places that do something special for ```eventSymbol.AssociatedField```.